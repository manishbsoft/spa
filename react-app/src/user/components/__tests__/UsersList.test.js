import React from 'react';

import UsersList from '../UsersList';
import UserItem from '../UserItem';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


configure({ adapter: new Adapter() });

describe("Users List Component", () => {
   

  it("should render Users Component", () => {  
    const wrapper = shallow(<UsersList />);
    expect(wrapper).toMatchSnapshot();
  });

  it("should render Users Component", () => {  
    const props = wrapper.props();
    const itemsLength = props.item && props.item.length;
    expect(wrapper.find(UserItem)).toEqual(itemsLength);
  });

});