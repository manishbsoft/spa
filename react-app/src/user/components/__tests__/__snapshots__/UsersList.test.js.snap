// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users List Component should render Users Component 1`] = `ShallowWrapper {}`;
